lambdas/src/main/kotlin/scorcerer/server/events/UserCreationEventHandler.kt

```diff
@@ -36,6 +45,31 @@ class UserCreationEventHandler : RequestHandler<SQSEvent, Unit> {
                     row[this.livePoints] = 0
                 }
             }
+
+            try {
+                transaction {
+                    LeagueTable.insert {
+                        it[this.name] = "Global"
+                        it[this.id] = "global"
+                    }
+                }
+            } catch (e: PSQLException) {
+                log.info("Global league exists: $e")
+            }
+
+            transaction {
+                LeagueMembershipTable.insert {
+                    it[this.memberId] = userCreationEvent.id
+                    it[this.leagueId] = leagueId
```

Comment:
I think this is wrong, `leagueId` refers to the column on the table. I assume you want `global` here